src/background.js

```diff
@@ -112,10 +112,25 @@ function migrateVersion(prevVersion,currentVersion){
   localStorage.currentVersion = currentVersion;
   addSubscription(prevVersion);
   updateTabList();
+  migrateDisabledSites();
   migrateBlockedDomains();
   migrateCookieBlockList();
 }
 
+
+/**
+ * migrates disabled sites from array data structure to object
+ */
+function migrateDisabledSites() {
+  
+   var obj = {};
+   var sites = JSON.parse(localStorage.disabledSites);
+  for(var i = 0; i < sites.length; i++){
```

Comment:
Here are a few better ways.
http://stackoverflow.com/questions/26264956/convert-object-array-to-hash-map-indexed-by-an-attribute-value-of-the-object
http://stackoverflow.com/questions/28835262/convert-array-to-hashmap-key-value-pair-javascript